dapp/src/utils/cardano/types.ts

```diff
@@ -0,0 +1,43 @@
+import { Token } from "./token";
+
+export type CardanoWallets = "eternl" | "flint" | "nami" | "typhoncip30";
+
+export type CardanoWalletInfo = {
+  name: string;
+  icon: string;
+  key: CardanoWallets;
+  url: string;
+};
+
+type ProjectedNftCardanoEventsResponseObject = {

```

Comment:
Do we need this? The Carp client library contains the Typescript definition of the type of all its endpoints

Reply:
Yeah now that the Carp PR has been merged I can do it.